From c3bf8a161d32916cdaac449ab9fd6c2a15a39391 Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Wed, 18 Jun 2025 16:34:43 +0300
Subject: [PATCH] [JSW-652] dts: jgs: fix uart address

Change uart address from fs1 to jgs
according to mwm_ww19_4 specs.

Change-Id: Ie1626fd75424e7b2ea5901b410d8e2a27b38d8df
Signed-off-by: Tal Yacobi <tyacobi@habana.ai>
---

diff --git a/dts/arc/jgs_emb.dtsi b/dts/arc/jgs_emb.dtsi
index efe3e62..c514ab5 100644
--- a/dts/arc/jgs_emb.dtsi
+++ b/dts/arc/jgs_emb.dtsi
@@ -484,10 +484,10 @@
 			label = "PINMUX";
 		};
 
-		uart0: uart@f9c8e000 {
+		uart0: uart@f866e000 {
 			compatible = "arm,cmsdk-uart";
 			clocks = <&refclk>;
-			reg = <0xf9c8e000 0x1000>;
+			reg = <0xf866e000 0x1000>;
 			interrupts = <59 0>; /* TODO */
 			node_label = "UART_0";
 			label = "UART_0";
